src/rpcnet.cpp

```diff
@@ -97,7 +97,7 @@ Value getpeerinfo(const Array& params, bool fHelp)
             "    \"pingtime\": n,             (numeric) ping time\n"
             "    \"pingwait\": n,             (numeric) ping wait\n"
             "    \"version\": v,              (numeric) The peer version, such as 7001\n"
-            "    \"subver\": \"/Satoshi:0.8.5/\",  (string) The string version\n"
+            "    \"subver\": \"/Classic:0.8.5/\",  (string) The string version\n"
```

Comment:
How will the RPC client react to this? will it break compatibility with core client?